### PR TITLE
fix: ensure circle elements in SVGs get currentColor fill

### PR DESCRIPTION
**Pull request by adamstankiewicz**
## Description

> Hi All, I'm seeing some people without heads in our icon library:

image

It looks our svgr config for the icons build process doesn't add `fill="currentColor` to <circle> elements in the SVGs, like it does for `<path>` elements. Implication: any icon with a circle(s) is likely currently missing the circle(s) when rendered.

### Deploy Preview

https://deploy-preview-2687--paragon-openedx.netlify.app/foundations/icons

## Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [ ] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [ ] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [ ] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [ ] Were your changes tested in the `example` app?
* [ ] Is there adequate test coverage for your changes?
* [ ] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [ ] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [ ] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

File: icons/.svgrrc.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ module.exports = {
         [
           "@svgr/babel-plugin-add-jsx-attribute",
           {
-            "elements": ["path"],
+            "elements": ["path", "circle"],
```
Comment (adamstankiewicz) [inline icons/.svgrrc.js:59]: This is the actual fix here :)

**Review by kalebjdavenport — APPROVED** [review]
I was nervous when I first opened the PR haha

File: icons/jsx/index.jsx [hunk 2]
```diff
@@ -472,7 +472,6 @@ export { default as Check } from "./Check";
 export { default as CheckBoxIcon } from "./CheckBoxIcon";
 export { default as CheckBoxOutlineBlank } from "./CheckBoxOutlineBlank";
 export { default as CheckCircle } from "./CheckCircle";
-export { default as CheckCircleLightOutline } from "./CheckCircleLightOutline";
```
Comment (viktorrusakov) [inline icons/jsx/index.jsx:475]: I assume this in not intended? (same comment for `icons/es5/index.js` file)

I think someone manually added jsx and js files for this icon but did not add the actual svg file, so `npm run build` removes this this import...

**Review by brian-smith-tcril — APPROVED** [review]
:shipit:

File: icons/jsx/index.jsx (hunk 2 above)
Comment (adamstankiewicz) [reply icons/jsx/index.jsx:475]: @viktorrusakov You are correct. Unfortunately, these lines will likely continue to be added until the appropriate SVG file is added...

Comment (viktorrusakov) [reply icons/jsx/index.jsx:475]: Yeah, I'm just worried about making a breaking change here in this PR. Can you maybe manually remove these changes, so we can safely merge this? And then we create a separate issue to add the missing SVG file.

Comment (brian-smith-tcril) [reply icons/jsx/index.jsx:475]: if we get https://github.com/openedx/paragon/pull/2869 merged we don't need to worry about this being a breaking change

Comment (viktorrusakov) [reply icons/jsx/index.jsx]: @brian-smith-tcril Yes, you're right, I merged it so there shouldn't be any problems now.

I took the liberty of updating this PR myself by rebasing it on `master` as I know @adamstankiewicz is quite busy with enterprise work and might not get the chance to get to this PR soon. I validated that running `npm run build` does not remove any icons anymore and that the fix works (this is one of the fixed icons that Adam added to the PR description)
image

I'm going to merge it in as it is quite an important fix and PR has been open for a long time now.

**Review by viktorrusakov — APPROVED** [review]

**Pull request merged**